package.json

```diff
@@ -37,12 +37,11 @@
   "dependencies": {
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
-    "@fortawesome/vue-fontawesome": "^2.0.6",
+    "@nuxtjs/fontawesome": "^1.1.2",
```

Comment:
Mettre `@nuxtjs/fontawesome` dans les `devDependencies`

> Instead of adding to dependencies inside package.json, use devDependencies (yarn add --dev or npm install --save-dev) 
cf. https://nuxtjs.org/docs/configuration-glossary/configuration-modules/

Reply:
Je me demande si on ne devrait pas déplacer la majorité des `dependencies` du projet dans `devDependencies` mais je ne suis pas du tout sûr des impacts... Vu que tout est compilé par la suite (mode static) je ne suis pas sûr qu'il y ait de réel impact.

Reply:
Je pense qu'il serait effectivement bon de déplacer d'autres paquets dans `devDependencies`, mais nous ferons ça dans une autre PR, d'accord ?

Reply:
Est-ce que je peux délayer le déplacement de ce paquet dans cette autre PR ?

Reply:
Oui, 👌 pour moi